hack/arktos-cni.rc

```diff
@@ -170,6 +203,10 @@ if [[ -n "${ARKTOS_NO_CNI_PREINSTALLED}" ]]; then
 elif [ "${CNIPLUGIN}" == "bridge" ]; then
     echo "cni plugin is bridge; arktos will use bridge to provision pod network"
     install_cni_bridge
+elif [ "${CNIPLUGIN}" == "mizar" ]; then
+    echo "cni plugin is mizar; arktos will use mizar to provision pod network"
+    install_arktos_containerd
```

Comment:
Instead of modifying install_arktos_containerd, can we have install_mizar_arktos_containerd for now so as to not break existing arktos? At some point both should become the same.
@yb01 What are your thoughts on this?

Reply:
Sure, we will create a new function ```install_mizar_arktos_containerd``` for installing arktos-containerd.